### ISSUE TITLE
Support for v10

### ISSUE THREAD

**Issue by sahilpurav** [aitboudad/ngx-loading-bar#161]
After upgrading to Angular v10. I've started getting the following error:

`npm WARN @ngx-loading-bar/core@5.0.0 requires a peer of tslib@^1.7.1 but none is installed. You must install peer dependencies yourself.`

Also, it has stopped working as soon as I upgraded to Angular v10

**Comment by aitboudad** [comment]
fixed in [v5.0.1](https://github.com/aitboudad/ngx-loading-bar/releases/tag/v5.0.1)

**Comment by davidkudera** [comment]
Hello, is there any plan for updating the other packages too?

```bash
npm WARN @ngx-loading-bar/http-client@5.1.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
npm WARN @ngx-loading-bar/router@5.1.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by milenon** [comment]
I'm receiving the following error:

`    ERROR in node_modules/@ngx-loading-bar/http-client/http.module.d.ts:1:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.
    
    This likely means that the library (@ngx-loading-bar/http-client) which declares LoadingBarHttpClientModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so.
 Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.
`

**Comment by rala72** [comment]
> Hello, is there any plan for updating the other packages too?
> 
> ```shell
> npm WARN @ngx-loading-bar/http-client@5.1.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
> npm WARN @ngx-loading-bar/router@5.1.0 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
> ```

I'm getting the same warning and have installed current version of tslib 2.0.3.

**Comment by fsmorygo** [comment]
I've created a pull request https://github.com/aitboudad/ngx-loading-bar/pull/180
Seems like it works, all tests passes.

**Comment by aitboudad** [comment]
fixed in [v5.1.1](https://github.com/aitboudad/ngx-loading-bar/releases/tag/v5.1.1), special thanks to @fsmorygo!
